### ISSUE TITLE
Preview & Load not dismissed when Load is pressed

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2989]
## tl;dr

A bug with the Preview & Load screen appeared at commit a5ee8af such that a load operation is initiated when the **Load** button is pressed but the screen does not close until the load completes.

## Details

I confirmed via binary search that this bug first appeared at commit a5ee8af (#2955).

As shown in the attached video, when I tee up a large set of files in Preview & Load and press the **Load** button, the load does begin (you can see the tab appear in the background). But since I'm accustomed to the screen being dismissed and watching the filling load thermometer, I ended up hitting the button again multiple times thinking I maybe mis-clicked and it ends up initiating repeated additional loads of the same data to even more pools.

https://github.com/brimdata/zui/assets/5934157/36d84e71-f2dd-4573-88f2-675848fd6a99

**Comment by philrz** [comment]
Verified in Zui commit 7dd553b.

As shown in the attached video, now when I click the **Load** button the Preview & Load screen is immediately dismissed and I can watch the rest of the load proceed by watching the thermometer fill up on the pool summary page.

https://github.com/brimdata/zui/assets/5934157/0d1779ed-324f-4b94-a2f7-3e51feae581f

Thanks @jameskerr!
